### PR TITLE
Add seed data to populate MongoDB

### PR DESCRIPTION
**Pull request by zsoltime**
I've just added two seed files to populate MongoDB with some sample data, using the schema I mentioned in #40.

I've already imported the sample documents to our database.

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -1,15 +0,0 @@
-{
-  "users": [{
-    "id": "",
-    "email": "",
-  }],
-  "posts": [{
-    "id": "",
-    "author": {
-      "name": "",
-
```
Comment (Heyjp) [inline schema.json:10]: Please remove the extra new line.

**Review by Heyjp — CHANGES_REQUESTED** [review]
Could you just update the schema.json new line and then we can merge.

**Comment by pchalubinska** [comment]
Isnt the requested change with the extra line in the deleted part since it's on red?

File: schema.json (hunk 1 above)
Comment (zsoltime) [reply schema.json:10]: @pchalubinska is right, I can't remove this as I deleted the whole `schema.json` file already.

Comment (Heyjp) [reply schema.json:10]: My mistake then. Will approve. Thanks @zsoltime @pchalubinska .

**Review by Heyjp — APPROVED** [review]

**Pull request merged**